project_euler/problem_34/sol1.py

```diff
@@ -0,0 +1,58 @@
+"""
+145 is a curious number, as 1! + 4! + 5! = 1 + 24 + 120 = 145.
+
+Find the sum of all numbers which are equal to the sum of the factorial of their digits.
+
+Note: As 1! = 1 and 2! = 2 are not sums they are not included.
+
+"""
+
+
+def factorial(n: int) -> int:
+    """
+    Returns the factorial of n
+    >>> factorial(5)
+    120
+    >>> factorial(1)
+    1
+    >>> factorial(0)
+    1
+    """

```

Comment:
Please and tests for `factorial(-1)` and `factorial(1.1)` and make sure the results match `math.factorial()`.

Reply:
Ok will do that

Reply:
I applied the changes. Please review it sir.